ambry-messageformat/src/main/java/com.github.ambry.messageformat/BlobAll.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.github.ambry.messageformat;
+
+import com.github.ambry.store.StoreKey;
+import java.nio.ByteBuffer;
+
+
+public class BlobAll {
```

Comment:
No, I can remove any getBlobAll related code. I thought it might be useful to have in the future, but I can always package these changes into a separate PR.